package.json

```diff
@@ -36,8 +36,9 @@
   "dependencies": {
     "jquery": "^3.1.1",
     "lodash": "^4.17.4",
-    "prototype": "timrichd/prototype.node.js",
+    "prototype": "github:timrichd/prototype.node.js",
```

Comment:
Does this mean that we're reliant on an ex employee not removing their github account?

Reply:
Looks like it ye, I'll see if I can get a separate repo set up and fork Tim's repo.

Reply:
But if only the author had actually updated npm with the latest version of that package... https://github.com/Rixius/prototype.node.js/issues/5

Reply:
We now have a public fork at: Congenica/prototype.node.js